controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -872,14 +872,20 @@ func (r *BareMetalHostReconciler) registerHost(prov provisioner.Provisioner, inf
 	}
 
 	// Create the hostFirmwareSettings resource with same host name/namespace if it doesn't exist
+	// Create the hostFirmwareComponents resource with same host name/namespace if it doesn't exist
 	if info.host.Name != "" {
 		if !info.host.DeletionTimestamp.IsZero() {
 			info.log.Info(fmt.Sprintf("will not attempt to create new hostFirmwareSettings in %s", info.host.Namespace))
+			info.log.Info(fmt.Sprintf("will not attempt to create new hostFirmwareComponents in %s", info.host.Namespace))
```

Comment:
Let's only log ones ("... create new hostFirmwareSettings and hostFirmwareComponents")